src/components/MyProfileActivityList/index.js

```diff
@@ -3,164 +3,178 @@ import PropTypes from 'prop-types';
-    };
+    // Gets the membership and involvement privacy
+    const { Membership, InvolvementPrivacy } = this.props;
+    // Style of privacy text
     const toggleTextStyle = {
       fontSize: '12pt',
     };
+    // Creates the opacity of the card to reflect the membership's privacy
     const membershipItemStyle = {
       opacity: Membership.Privacy || InvolvementPrivacy ? '0.5' : '1',
     };
     // The Grid lengths for each content inside of the card based on Material-UI's breakpoints
     const cardContentLengths = {
-      text: {
-        xs: 7,
-        sm: 8,
-        md: 6,
-        lg: 7,
-        xl: 6,
+      contentOne: {
+        xs: 8,
+        sm: 9,
+        md: 9,
+        lg: 8,
+        xl: 9,
       },
-      privacy: {
-        xs: 2,
-        sm: 2,
-        md: 2,
-        lg: 2,
-        xl: 2,
-      },
-      picture: {
-        xs: 3,
-        sm: 2,
-        md: 2,
-        lg: 3,
-        xl: 2,
+      contentTwo: {
+        xs: 4,
+        sm: 3,
+        md: 3,
+        lg: 4,
+        xl: 3,
       },
     };
 
-    // If the Involvement is a regular (non-special/secret group - AKA Public) it is False.
     let myProfileInvolvementsList;
+
+    // If the Involvement is a regular (non-special/secret group - AKA Public) it is False.
     if (!InvolvementPrivacy) {
       myProfileInvolvementsList = (
         <div>
-          <Grid container alignItems="center" justify="center">
-            <Grid
-              item
-              xs={cardContentLengths.text.xs}
-              sm={cardContentLengths.text.sm}
-              md={cardContentLengths.text.md}
-              lg={cardContentLengths.text.lg}
-              xl={cardContentLengths.text.xl}
-            >
-              <List>
-                <ListItem style={{ paddingLeft: '0.5rem' }}>
-                  {/* A link to the activity is only available if the user is online */}
-                  {this.props.network === 'online' ? (
-                    <Link
-                      className="gc360-link"
-                      to={`/activity/${Membership.SessionCode}/${Membership.ActivityCode}`}
-                      style={membershipItemStyle}
-                    >
-                      <Typography>
-                        <b>{Membership.ActivityDescription}</b>
-                      </Typography>
-                      <Typography>{Membership.SessionDescription}</Typography>
-                      <Typography>{Membership.ParticipationDescription}</Typography>
-                    </Link>
-                  ) : (
-                    <div style={membershipItemStyle}>
-                      <Typography>
-                        <b>{Membership.ActivityDescription}</b>
-                      </Typography>
-                      <Typography>{Membership.SessionDescription}</Typography>
-                      <Typography>{Membership.ParticipationDescription}</Typography>
-                    </div>
-                  )}
-                </ListItem>
```

Comment:
This code wasn't removed, it's moved to another line to Github supposedly thinks that it's all gone